tests/net/webrequest.cpp

```diff
@@ -186,11 +186,11 @@ TEST_CASE_METHOD(RequestFixture,
     if ( !InitBaseURL() )
         return;
 
-    Create("/bytes/65536");
+    Create("/bytes/1000");
```

Comment:
I changed this to 1000, because go-httpbin doesn't add a `content-length` header when the size is over 2048 bytes. Resulting in the functions below to return `-1`.